source/guides/stacks/dlrs.rst

```diff
@@ -20,16 +20,17 @@ customized solutions, and enables you to quickly prototype and deploy Deep
 Learning workloads. Use this guide to run benchmarking workloads on your
 solution.
 
-The latest release of the Deep Learning Reference Stack (`DLRS V5.0`_ ) supports the following features:
+The latest release of the Deep Learning Reference Stack (`DLRS V6.0`_ ) supports the following features:
 
-* TensorFlow* 1.15 and TensorFlow* 2.0, an end-to-end open source platform for machine learning (ML).
+* TensorFlow* 1.15 and TensorFlow* 2.2.0, an end-to-end open source platform for machine learning (ML).
 * PyTorch* 1.3, an open source machine learning framework that accelerates the path from research prototyping to production deployment.
 * PyTorch Lightning* which is a lightweight wrapper for PyTorch designed to help researchers set up all the boilerplate state-of-the-art training.
-* Transformers* , a state-of-the-art Natural Language Processing (NLP) for TensorFlow 2.0 and PyTorch.
-* Intel® OpenVINO™ model server version 2019_R3, delivering improved neural network performance on Intel processors, helping unlock cost-effective, real-time vision applications.
+* Transformers* , a state-of-the-art Natural Language Processing (NLP) for TensorFlow 2.0 and Flair for PyTorch.
```

Comment:
May be change it to:
* Transformers* , a state-of-the-art Natural Language Processing (NLP) library for TensorFlow 2.0 and PyTorch 
* Flair*, a PyTorch NLP framework

Reply:
good call -- they're not married. :)